functions/go/bind/pkg/meta/kinds.go

```diff
@@ -0,0 +1,90 @@
+// Copyright 2022 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package meta
+
+import (
+	"fmt"
+
+	"k8s.io/apimachinery/pkg/runtime/schema"
+)
+
+// IsClusterScoped returns true if the specified GVK is cluster scoped, or an error if this cannot be determined.
+func IsClusterScoped(gvk schema.GroupVersionKind) (bool, error) {
+	ki := findKindInfo(gvk)
+	if ki == nil {
+		return false, fmt.Errorf("kind %v not known", gvk)
+	}
+	return ki.ClusterScoped, nil
+}
```

Comment:
FYI, the go SDK has `IsClusterScoped` `IsNamespaceScoped` functions that also relies on the hard-coded namespace list https://github.com/GoogleContainerTools/kpt-functions-sdk/blob/master/go/fn/internal/namespace.go#L8. I'm happy to extend the list to include CNRM and change the IsClusterScoped to only require GVK args.